index.html

```diff
@@ -1,71 +1,102 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
-  <title>SpaceWalkers</title>
+    <title>SpaceWalkers</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
-  <link href="css/index.css" rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu"
+      rel="stylesheet"
+    />
+    <link href="css/index.css" rel="stylesheet" />
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
 
-<body>
+  <body>
+    <!-- navigation code here -->
+    <header class="top">
+      <div><h1>SpaceWalkers</h1></div>
 
- <!-- navigation code here -->
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contacts</a>
+      </nav>
+    </header>
 
-  <div class="container home">
+    <div class="container home">
+      <header class="intro">
+        <!-- image code here -->
+        <img class="pics" src="../Sprint/img/main-header.png" alt="main-header" />
 
```

Comment:
this image doesn't render, it appears the source is off. Usually when assigning the src it's relative to where your index.html sits. Try removing the `/Sprint`